files/typescript/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "typescript.js",
+  "repo": "Microsoft/TypeScript",
+  "files": {
+    "basePath": "build/",
```

Comment:
I dont see a build/ directory in the repo https://github.com/Microsoft/TypeScript/tree/v1.6.2